maidroid_tool/api.lua

```diff
@@ -52,13 +52,23 @@ function maidroid_tool.register_writer(nodename, options)
 		-- if time is positive, this node is active.
 		if time >= 0 then
 			if time <= duration then
-				meta:set_float("time", time + 1)
+				meta:set_float("time", time + elapsed)
 				meta:set_string("formspec", formspec.active(time))
 			else
 				meta:set_float("time", -1)
 				meta:set_string("output", "")
 				meta:set_string("formspec", formspec.inactive)
-				inventory:set_stack("main", 1, ItemStack(output))
+				local resultitem = ItemStack(output)
+
+				if output == "maidroid_core:custom" then
```

Comment:
Yes, this is makeshift, l used it to test if the pdisc mod works.
The custom core also needs to be writeable, else players have to make a new core every time they change/test/adjust their program.